Update README.md (#2331)
DataDog/integrations-extras: flume/README.md

   Agent's configuration directory to start collecting Flume performance data.
   See the [sample `flume.d/conf.yaml`][8] file for all available configuration options.

   This check has a limit of 350 metrics per instance. The number of returned metrics is indicated in the status output.
   You can specify the metrics you are interested in by editing the configuration below.

>    To learn how to customize the metrics to collect, see the [JMX Checks documentation][9] for detailed instructions.

Reviewer: ```suggestion
   For detailed instructions on customizing the metrics to collect, see the [JMX Checks documentation][9].
```
